pset-iterators.js

```diff
@@ -138,6 +170,15 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const livingPresidents = presidents.filter((president) => {
```

Comment:
@Ashbk32 great job, be careful with your indentation, line 174 should be indented one level to the right